### ISSUE TITLE
[SF][A11y][Card]: Support ariaLevel  in 0.131.x

### ISSUE THREAD

**Issue by ericchan1983** [SAP/ui5-webcomponents-react#2156]
**Describe the bug**
As the ticket mentioned, the new ariaLevel is exposed on ui5-card, could the react component Card support the new ariaLevel?
https://github.com/SAP/ui5-webcomponents/issues/3712
"However, ariaLevel property is now exposed on ui5-card level and can be used to achieve the same result. It will be available with the next version from branch 0.31. Keep in mind that once you upgrade the version of ui5-webcomponents you are using, you will no longer use the ui5-card property, but the one that is exposed on ui5-card-header"

**Isolated Example**
*Please provide an isolated example if possible by forking [this codesandbox](https://codesandbox.io/s/ui5-webcomponents-react-template-29l34).*

**To Reproduce**

**Expected behavior**
The title of the card should be shown as the configured aria-level.

**Screenshots**

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.31.18
`@ui5/webcomponents-react` version:0.131.20
Operating System: macOS BigSur 11.6
Browser: Google Chrome Version 93.0.4577.82 (Official Build) (x86_64)

**Additional context**

**Comment by MarcusNotheis** [comment]
Hey @ericchan1983,
you can use the HTML standard `aria-level` property here.

**Comment by ericchan1983** [comment]
Hi @MarcusNotheis,
As the screenshot shown, the aria-level of the card doesn't apply to the card header, the card header is still the aria-level="3".

![image](https://user-images.githubusercontent.com/1614528/134319422-f4d824f3-021e-40d6-93d6-35610074f312.png)

Regards,
Eric Chen

**Comment by fifoosid** [comment]
Hi guys, 

I created a [CodeSandBox example](https://codesandbox.io/s/friendly-visvesvaraya-g0odu) and everything works as expected

**Comment by Lukas742** [comment]
Hi @ericchan1983 

since `aria-level` is private it's not parsed as `ariaLevel` (camelCase) here, instead you need to set the default `aria-level` attribute as @fifoosid has shown in the codeSandbox.
I'm going to close this issue now, if there are still open questions please feel free to reopen it or create a new one.

Thanks @fifoosid!
